vertical-tabbrowser.xml

```diff
@@ -220,6 +191,21 @@
           draggedTab.style.transform = 'translateY(' + translateVal + 'px)';
           // console.log('draggedTab transform: ' + draggedTab.style.transform + ' should be ' + 'translateY(' + translateVal + 'px)');
 
+          let tabbrowser = document.getElementById('tabbrowser-tabs');
+          let scrollbox = document.getAnonymousElementByAttribute(tabbrowser, 'anonid', 'arrowscrollbox');
+          let scrollbuttonDown = document.getAnonymousElementByAttribute(scrollbox, 'anonid', 'scrollbutton-down');
+          let scrollbuttonUp = document.getAnonymousElementByAttribute(scrollbox, 'anonid', 'scrollbutton-up');
+          let mouseY = event.clientY;
+
+          if (mouseY > (document.getElementById('tabbrowser-tabs').boxObject.height + 10)){
```

Comment:
We already have `document.getElementById('tabbrowser-tabs')` in the `tabbrowser` variable above.  Let's use that.  😉